wsclient/src/main/java/org/hpccsystems/ws/client/utils/Utils.java

```diff
@@ -691,4 +700,30 @@ public static boolean currentOSisLinux()
           }
           throw new IllegalArgumentException(enumclass.getName() +".'" + strvalue + "' is not valid.");
       }
+      
+      public static EclInfo GetEcl(String content) {
```

Comment:
method name should start with lower case, but perhaps it can be renamed parseECL()?